docs/how-to/h-upgrade.md

```diff
@@ -0,0 +1,115 @@
+# How to upgrade between minor versions
+
+> **:warning: WARNING:** this is currently a feature in "edge" or "beta" risks channels. We do NOT recommend to use features in production that are not in the `stable` channel! Also, contact [Canonical Data Platform team](https://chat.charmhub.io/charmhub/channels/data-platform) if you are interested in the topic.
```

Comment:
**suggestion:** this wording isnt particularly clear to me. Given that the README should represent the state of the `stable` channel, could we rephrase to say something along the lines of 'only supported in stable'?

If I understood the intent.